rlang/src/test/java/org/apache/zeppelin/r/ShinyInterpreterTest.java

```diff
@@ -221,7 +221,7 @@ public void testInvalidShinyApp()
 
     resultMessages = context2.out.toInterpreterResultMessage();
     assertTrue(resultMessages.get(1).getData(),
-            resultMessages.get(1).getData().contains("object 'Invalid_code' not found"));
+            resultMessages.get(1).getData().contains("'Invalid_code'"));
```

Comment:
Why this change ?

Reply:
My system language is German. When I run tests locally, some tests fail because the JVM translates the error message into German. With commit https://github.com/apache/zeppelin/pull/3925/commits/ea099abef02a0b971c863c1bb1b70aa149bb74fc I have removed the part that is translated by the JVM.